packages/hash/frontend/src/components/layout/PageHeader/SearchBar/index.tsx

```diff
@@ -155,34 +188,56 @@ const SearchBarWhenSearchIsEnabled: React.VFC = () => {
         },
         position: "relative",
         height: "100%",
-        ...(isMobile && displaySearchInput
-          ? {
-              position: "absolute",
-              width: "100%",
-              zIndex: 1,
-              left: 0,
-              top: theme.spacing(1.5),
-              px: 2,
-            }
-          : {}),
+        ...getSearchBarResponsiveStyles(isMobile, displaySearchInput),
       }}
       ref={rootRef}
     >
-      {!isMobile ? (
-        <DesktopSearch
-          displayedQuery={displayedQuery}
-          setQueryText={setQueryText}
-          setResultListVisible={setResultListVisible}
-        />
+      {/* If the user is in mobile view and the search icon isn't clicked, display the icon */}
+      {isMobile && !displaySearchInput ? (
+        <IconButton
+          sx={{
+            height: "32px",
+            width: "32px",
+            display: "flex",
+            alignItems: "center",
+          }}
+          onClick={() => setDisplaySearchInput(true)}
+        >
+          <SearchIcon sx={{ height: "16px", width: "auto" }} />
+        </IconButton>
       ) : (
-        <MobileSearch
-          displayedQuery={displayedQuery}
-          setQueryText={setQueryText}
-          setResultListVisible={setResultListVisible}
-          displaySearchInput={displaySearchInput}
-          setDisplaySearchInput={setDisplaySearchInput}
-        />
+        <Box
+          sx={{
+            display: "flex",
+            alignItems: "center",
+            width: "100%",
+            background: "white",
+            zIndex: 1,
+          }}
+        >
+          <SearchInput
+            displayedQuery={displayedQuery}
+            isMobile={isMobile}
+            setQueryText={setQueryText}
+            setResultListVisible={setResultListVisible}
+          />
+
+          {isMobile && (
+            <Button
+              onClick={() => {
+                setQueryText("");
+                setDisplaySearchInput(false);
+              }}
+              variant="tertiary_quiet"
+              size="xs"
```

Comment:
Ideally the height would be unset so that it matches the height of the input component exactly, but this is fine.

Reply:
I had tried an unset height, but it looks odd on a mobile screen. The design has size set as `xs` as well, so just used that.